Gemfile

```
@@ -29,3 +29,8 @@ end
 group :test do
   gem "rake"
 end
+
+group :coverage, optional: ENV['COVERAGE']!='yes' do
```

I think Rubocop would want:
```suggestion
group :coverage, optional: ENV['COVERAGE'] != 'yes' do
```

---

updated